grpcService/pkg/entities/requests.go

```diff
@@ -44,3 +44,15 @@ type DeleteUserRequest struct {
 type DeleteUserResponse struct {
```

Comment:
entities pkg doesn't need to represent anything about request/response, this belongs to the domain pkg, one example of entity is User structure, and a UpdateUserRequest will have a field User